programs/tests/playTestsMSYS.sh

```diff
@@ -0,0 +1,190 @@
+#!/bin/sh -e
```

Comment:
playTests.sh doesn't work with MSYS on Windows because of "diff" and some minor things. playTestsMSYS.sh is slightly changed playTests.sh which works with MSYS.